File: 2310817120009_AlysaArmelia_PemrogramanI_Modul3/Python/PRAK305-2310817120009-AlysaArmelia.py
hari = 0
jam = 0
menit = 0
detik = 0

input_dimasukkan = int(input())

if input_dimasukkan >= 86400:
    hari = input_dimasukkan // 86400
    input_dimasukkan %= 86400

if input_dimasukkan >= 3600:
    jam = input_dimasukkan // 3600
    input_dimasukkan %= 3600

if input_dimasukkan >= 60:
    menit = input_dimasukkan // 60
    input_dimasukkan %= 60

detik = input_dimasukkan
if hari > 0:
    print(f"{hari} hari {jam:02d}:{menit:02d}:{detik:02d}")
else:
    print(f"{jam:02d}:{menit:02d}:{detik:02d}")

File: 2310817120009_AlysaArmelia_PemrogramanI_Modul3/Bahasa C/PRAK305-2310817120009-AlysaArmelia.c
#include <stdio.h>

int main()
{
    int diketahui;
    int hari =0;
    int jam = 0;
    int menit = 0;
    int detik = 0;

    scanf("%d", &diketahui);
    
    hari = diketahui / 86400;
    diketahui %= 86400;

    jam = diketahui/ 3600;
    diketahui %= 3600;
    
    menit = diketahui / 60;
    diketahui %= 60;

    detik = diketahui / 1;
    diketahui %= 1;

    if (hari > 0){
       printf("%d hari %02d:%02d:%02d",hari , jam , menit , detik); 
    }
    else if (jam > 0){
       printf("%02d:%02d:%02d", jam , menit , detik); 

    }else if (menit > 0){
       printf("%02d:%02d:%02d", jam , menit , detik); 
       
    }else if (detik > 0){
       printf("%02d:%02d:%02d", jam , menit , detik); 
    }
return 0;
}